File: Algorithms/Programs/JavaScript/FactorialRecursion.js
// Factorial using recursion
function factorial(num) 
{
    if (num >= 0) 
    {
        if (num == 1 || num == 0) 
        {
            return 1;
        }
        let fact = num * factorial(num - 1);
        return fact;
    }

    else return undefined;
}

console.log(factorial(any_number_here));

File: Algorithms/Programs/Python/factorial_recursion.py
# Python program to find the factorial of a number using recursion

def factorial(x):
    if x == 1 or x == 0:
        return 1
    else:
        return (x * factorial(x-1))

num = int(input("Enter a number: "))

result = factorial(num)
print("The factorial of", num, "is", result)
